### ISSUE TITLE
Creating account on ejabberd with CAPTCHA enabled does not work

### ISSUE THREAD

**Issue by crosser** [movim/movim#768]
I have a movin instance backed by ejabberd on which account creation is behind CAPTCHA:

```
  mod_register:
  ##
  ## Protect In-Band account registrations with CAPTCHA.
  ##
    captcha_protected: true
  ##
  ## Set the minimum informational entropy for passwords.
  ##
    password_strength: 32
  ##
  ## After successful registration, the user receives
  ## a message with this subject and body.
  ##
    welcome_message:
      subject: "Welcome!"
      body: |-
        Hi.
        Welcome to this XMPP server.
  ##
  ## When a user registers, send a notification to
  ## these XMPP accounts.
  ##
    registration_watchers:
      - "crosser@average.org"
  ##
  ## Only clients in the server machine can register accounts
  ##
  ##   ip_access: trusted_network
  ##
  ## Local c2s or remote s2s users cannot register accounts
  ##
    access_from: deny
    access: register
```

It works, I can successfully register and account e.g. via Conversations. But when I try to register via Movim, captcha html content seems to be unnecessarily escaped:
![movim-captcha-hwh3uty9_o](https://user-images.githubusercontent.com/1404921/50538477-b3112a00-0b70-11e9-8376-02e420077010.png)

and of course registration is impossible.

**Comment by edhelas** [comment]
Something regarding this has been fixed yesterday. Are you using trunk?

**Comment by crosser** [comment]
I am using 0.14 release on this pod. I will try trunk, thanks.

**Comment by crosser** [comment]
Yes, commit e990b4b fixes the escaping issue.

It is still impossible to register, because you do not have captcha on the page and have to open it in a separate window/tab. And there is no chance to do it in 30 seconds after which you get redirected back to login page.

**Comment by edhelas** [comment]
Yes, this is another issue.

**Comment by crosser** [comment]
fixed by e990b4b
